test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -3212,6 +3212,15 @@ private void insertGroupEx(String lastModified, String groupName,
     }
   }
 
+  private void deleteGroup(String groupName) throws SQLException {
+    executeUpdate(String.format(
+        "delete from dm_group where group_name = '%s' and r_object_id = '%s'",
```

Comment:
Why include group_name and user_name here?

Reply:
I get why there are two delete statements. I don't see why you are including a condition on both group_name and r_object_id in one query, and both user_name and r_object_id in the other query. It seems like name or object ID should do the trick. Both should correctly identify the rows we want to delete.